src/MoBi.CLI/MoBi.CLI.csproj

```diff
@@ -23,16 +23,18 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="9.0.6" />
     <PackageReference Include="CommandLineParser" Version="2.9.1" />
-    <PackageReference Include="OSPSuite.Core" Version="13.0.47" />
-    <PackageReference Include="OSPSuite.Presentation" Version="13.0.47" />
-    <PackageReference Include="OSPSuite.Assets" Version="13.0.47" />
+    <PackageReference Include="OSPSuite.Core" Version="13.0.49" />
+    <PackageReference Include="OSPSuite.Presentation" Version="13.0.49" />
+    <PackageReference Include="OSPSuite.Assets" Version="13.0.49" />
     <PackageReference Include="OSPSuite.Utility" Version="4.1.1.3" />
     <PackageReference Include="OSPSuite.FuncParser" Version="4.0.0.74" GeneratePathProperty="true" />
     <PackageReference Include="OSPSuite.SimModel" Version="4.0.0.76" GeneratePathProperty="true" />
     <PackageReference Include="OSPSuite.SimModelSolver_CVODES" Version="4.1.0.19" GeneratePathProperty="true" />
+    <PackageReference Include="SixLabors.ImageSharp" Version="2.1.11" />
     <PackageReference Include="System.Data.SQLite.Core" Version="1.0.119" GeneratePathProperty="true" />
-    <PackageReference Include="OSPSuite.Infrastructure" Version="13.0.47" />
-    <PackageReference Include="OSPSuite.Infrastructure.Castle" Version="13.0.47" />
+    <PackageReference Include="OSPSuite.Infrastructure" Version="13.0.49" />
+    <PackageReference Include="OSPSuite.Infrastructure.Castle" Version="13.0.49" />
+    <PackageReference Include="System.Drawing.Common" Version="9.0.8" />
```

Comment:
This has to be added in MoBi and in MoBi.CLI to make sure that the versions match when resolved